Gemfile

```diff
@@ -57,6 +57,8 @@ group :development, :test do
   gem "rubocop-performance", require: false
   gem "brakeman", require: false
   gem "toxiproxy", "~> 1.0.0"
+  gem "better_errors"
+  gem "binding_of_caller"
```

Comment:
Does this work on JRuby (without platform constraint)? I have seen some testing done by @kares in this repo on JRuby and it will be nice to keep it possible to test on JRuby locally.

Reply:
gem *binding_of_caller* won't work on JRuby as it has a C extension

Reply:
Would `platforms: :mri` help here?